docs/flush-tables-with-read-lock.md

```diff
@@ -22,18 +25,16 @@ Long-running queries with `FLUSH TABLES WITH READ LOCK` enabled can leave the se
 
 !!! note
    
-    All operations described in this section have no effect when [backup locks](https://docs.percona.com/percona-server/8.1/backup-locks.html) are used.
+    All operations described in this section have no effect when [Backup locks] are used.
```

Comment:
missing the link?

Reply:
I've applied a new format for links here.